### PR TITLE
Make generated apks contain version name and code

### PR DESCRIPTION
**Pull request by danielgomezrico**
Its cool to know which version contains the apk easy.

### CODE REVIEW

**Comment by vanniktech** [comment]
Don't you have to use `$` to reference the version name and codes?

File: app/build.gradle [hunk 1]
```diff
@@ -25,6 +25,9 @@ android {
         versionCode versions.code // Notice that you may want to use BUILD_NUMBER from CI in real project with own CI.
         versionName versions.name
 
+        // Make generated apks contain version name and code
+        setProperty("archivesBaseName", "-${versions.name}-${versions.code}")
```
Comment (artem-zinnatullin) [inline app/build.gradle]: Is there a way to do this without `setProperty()`? Like just `archivesBaseName = "-${versions.name}-${versions.code}"`?

Comment (danielgomezrico) [inline app/build.gradle]: @artem-zinnatullin yeah, check next coomit, looks more clean.

**Comment by lenguyenthanh** [comment]
👍

File: app/build.gradle [hunk 2]
```diff
@@ -25,6 +25,9 @@ android {
         versionCode versions.code // Notice that you may want to use BUILD_NUMBER from CI in real project with own CI.
         versionName versions.name
 
+        // Make generated apks contain version name and code
+        project.archivesBaseName = "app-${versions.name}-${versions.code}"
```
Comment (vanniktech) [inline app/build.gradle]: Do we need the `project`?

**Comment by vanniktech** [comment]
👍

File: app/build.gradle [hunk 3]
```diff
@@ -25,6 +25,9 @@ android {
         versionCode versions.code // Notice that you may want to use BUILD_NUMBER from CI in real project with own CI.
         versionName versions.name
 
+        // Make generated apks contain version name and code
```
Comment (artem-zinnatullin) [inline app/build.gradle]: nit: `.` at the end http://nedbatchelder.com/blog/201401/comments_should_be_sentences.html (learned that from PRs to Square projects)

**Comment by artem-zinnatullin** [comment]
👍 (checked locally)

File: app/build.gradle [hunk 4]
```diff
@@ -25,6 +25,9 @@ android {
...
+        archivesBaseName = "app-${versions.name}-${versions.code}"
```
Comment (artem-zinnatullin) [inline app/build.gradle]: Now that looks great!

File: app/build.gradle (hunk 3 above)
Comment (danielgomezrico) [inline app/build.gradle]: Agree

Comment (artem-zinnatullin) [inline app/build.gradle]: Should be "Agree." lol

**Comment by danielgomezrico** [comment]
Guys why codecov failed?

**Comment by Plastix** [comment]
@danielgomezrico It might just be an issue with codecov. The check on my PR is failing too.

**Comment by artem-zinnatullin** [comment]
It fine, diffing code coverage between branches is not trivial.

**Comment by artem-zinnatullin** [comment]
@danielgomezrico thanks!

**Pull request merged**